tfjs-backend-webgpu/src/kernels/conv2d_mm_vec4_webgpu.ts

```diff
@@ -238,4 +242,152 @@ export class Conv2DMMVec4Program implements WebGPUProgram {
       `;
     return userCode;
   }
+
+  getSampleAWithRemainderWgsl(index: number): string {
```

Comment:
Can you add a comment above the line to explain that the `index` is to avoid repeated definition error?

Reply:
This is from GLSL, https://github.com/tensorflow/tfjs/blob/master/tfjs-backend-webgpu/src/kernels/Conv2D.ts#L59